airqo-platform/AirQo-api
# Device registry bug fix

- What does this PR do?
Fixed database configuration issues, temporarily disabled IoT core endpoints as I look into [this issue](https://github.com/airqo-platform/AirQo-api/issues/37#issue-547315622)
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/APA-112?atlOrigin=eyJpIjoiNzdjMmU1MWVmODdmNDJiZWIxYjEwNjM2ZDU3MDI0ODkiLCJwIjoiaiJ9).
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/device-registry` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run test` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
base URI: https://device-registry-dot-airqo-250220.appspot.com
partial URI: /api/v1/devices
More details about how to run this end of creating a device can be found [here](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=897606332)
